releasenotes/notes/0.17/add-parameters-and-is_parameterized-to-pulse-d5695dd14e407537.yaml

```diff
@@ -0,0 +1,25 @@
+---
+features:
```

Comment:
Can this be removed if https://github.com/Qiskit/qiskit-terra/pull/5854 is merged? The parameter framework of pulse module will be updated by this PR (it is centralized in manager class, rather than attaching parameter table to each piece of the program).

In https://github.com/Qiskit/qiskit-terra/pull/5854 there is also deprecation statement regarding this reno. That statement can be removed along with this. @lcapelluto ?

Reply:
I thought `.parameters` and `is_parameterized` was still supported on these objects?